cou/cli.py

```diff
@@ -241,5 +242,12 @@ def entrypoint() -> None:
         logger.error("Unexpected error occurred.")
         logger.exception(exc)
         sys.exit(2)
+    else:
+        completed_successfully = True
     finally:
+        if not completed_successfully:
+            logger.error(
+                "See the known issues at https://canonical-charmed-openstack-upgrader.readthedocs-"
+                "hosted.com/en/stable/reference/known-issues/"
+            )
```

Comment:
This will be print even if `Exception` happened or KeyboardInterrupt, which does not look good to me. Can we move it only for COUException?